includes/tracks/class-wc-site-tracking.php

```diff
@@ -83,14 +89,36 @@ public static function add_tracking_function() {
 	}
 
 	/**
-	 * Add empty tracking function to admin footer when tracking is disabled in case
-	 * it's called without checking if it's defined beforehand.
+	 * Adds a function to load tracking scripts and enable them client-side on the fly.
+	 * Note that this function does not update `woocommerce_allow_tracking` in the database
+	 * and will not persist enabled tracking across page loads.
 	 */
-	public static function add_empty_tracking_function() {
+	public static function add_enable_tracking_function() {
+		global $wp_scripts;
+		$woo_tracks_script = $wp_scripts->registered['woo-tracks']->src;
+
 		?>
 		<script type="text/javascript">
-			window.wcTracks = window.wcTracks || {};
-			window.wcTracks.recordEvent = function() {};
+			window.wcTracks.enable = function( callback = null ) {
```

Comment:
Ideally, I think this should be moved to a utils file like `lib/tracks.js` in wc-admin to take advantage and promise/polyfill support and avoid cluttering the footer further.

However, in the interest of time, this provides a way to quickly enable the scripts.